python/semantic_kernel/prompt_template/utils/template_function_helpers.py

```diff
@@ -29,6 +29,32 @@ def create_template_helper_from_function(
     base_arguments: "KernelArguments",
     template_format: TEMPLATE_FORMAT_TYPES,
     allow_dangerously_set_content: bool = False,
+    enable_async: bool = False,
+) -> Callable[..., Any]:
```

Comment:
Since we're here in this method, should we update the docstring? It'll help us on our journey to improve our docs.